doc/Reparenting.md

```diff
@@ -48,13 +46,12 @@ Larger Vitess deployments typically do implement semisynchronous replication.
 You can use the following <code>[vtctl]({% link reference/vtctl.md %})</code>
 commands to perform reparenting operations:
 
-* <code>[PlannedReparentShard](#plannedreparentshard:-planned-reparenting)</code>
-* <code>[EmergencyReparentShard](#emergencyreparentshard:-emergency-reparenting)</code>
+* <code>PlannedReparentShard</code>
+* <code>EmergencyReparentShard</code>
 
 Both commands lock the shard for write operations. The two commands
```

Comment:
Both commands lock the shard for write operations.
->
Both commands lock the Shard record in the global topology server.